Packs/ServiceNow/Scripts/ServiceNowAddComment/ServiceNowAddComment.py

```diff
@@ -0,0 +1,56 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    ticketID = demisto.args().get('ticketID', 'none')
+    note = demisto.args().get('note')
+    tag = demisto.args().get('tag')
+    table_name = demisto.args().get('table_name')
+    using = demisto.args().get('using')
+
+    command_args = {}
+
+    if ticketID == 'none':
```

Comment:
Please move all this logic to another function.
The main should be clean and contain only the args, the call to the function and return_results